articles/migrations/guides/extensibility-node12.md

```diff
@@ -0,0 +1,221 @@
+
+::: note
+Private Cloud and Managed Private Cloud (PSaaS) customers have had their tenants migrated already. No further action is necessary.
+:::
+
+### Important Dates
+
+* **2019 December 31**: [Node 8 is no longer under long-term support (LTS)](https://github.com/nodejs/Release#release-schedule)
+* **2020 April xx**: The Webtask runtime using Node 12 becomes available to Auth0 customers
+* **2020 April xx**: All official Auth0 Extensions will be updated to run on Node 12 and available for you to upgrade in the **Installed Extensions** tab of the [Extensions page](${manage_url}/#/extensions)
+* **2020 April xx**: Any tenants created after this date are already using the Node.js 12 runtime, and no action is required.
+* **2020 April xx**: Tenants with NO Extensibility code will be automatically be upgraded to use Node 12
+* **2020 April xx**: All Auth0 tenants on the Auth0 public cloud which have not already migrated to the Node.js 12 runtime, will be automatically migrated between April xx, 2020 and June xx, 2020
+* **2019 June xx**: Node 8 support in Auth0 is permanently removed
+
+## How to enable the Node 12 runtime
+
+Node 12 can be enabled through the new Extensibility panel on the [Advanced Tenant Settings](${manage_url}/#/tenant/advanced) page of the Dashboard.
+
+![Runtime toggle](/media/articles/migrations/node-runtime1.png)
+
+![Runtime toggle options](/media/articles/migrations/node-runtime2.png)
+
+::: warning
+Changing the runtime may break your existing Rules, Hooks, and Custom Database/Social Connections. We recommend that you first switch your development tenant to the Node 12 runtime, test your setup, and switch your production tenant only when you have identified there are no breaking changes.
+:::
+
+## Whitelist the new URLs
+
+The [Authorization Extension](/extensions/authorization), the [Delegated Administration Extension](/extensions/delegated-admin) and the [Single Sign-on (SSO) Dashboard Extension](/extensions/sso-dashboard) require whitelisting the URLs used to access extensions and custom webtasks. When you upgrade to Node 12, the URLs you use to access extensions and custom webtasks will change. This is a breaking change for these extensions.
+
+If you use any of these extensions, **you must whitelist the new URLs** both as Allowed <dfn data-key="callback">Callback</dfn> and as Allowed Logout URLs.
+
+The change is a `-node12` that is appended before the `webtask.io` part and a `prod-` that is appended before the region part. So if you accessed an extension using the URL `https://${account.tenant}.us8.webtask.io/dummy-extension-url`, when you upgrade to Node 12 the URL will be `https://${account.tenant}.prod-us-node12.webtask.io/dummy-extension-url`.
+
```

Comment:
Oh cool, so we already had some instructions for migrating some specific extensions to Node 8. These instructions may be slightly different this time around, and we will (hopefully) make it a little easier on the customer.

When we have a draft of these instructions ready (hopefully in the next day or so) we'll send them to you to review and incorporate them.

Reply:
Actually, no, I was wrong. This aspect of the configuration will need to be modified the same way.